### PR TITLE
Feat/storybook

### PR DESCRIPTION
**Pull request by JUtupe**

### CODE REVIEW

File: .github/workflows/storybook.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Build and Deploy
+on:
+  push:
```
Comment (DiD3n) [inline .github/workflows/storybook.yml:3]: where push

File: .storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions"
+  ],
+  framework: "@storybook/react",
+  core: {
+    builder: "@storybook/builder-vite"
```
Comment (DiD3n) [inline .storybook/main.js:31]: yoo vite support 
didn't expect this

File: .storybook/preview-head.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
+    body, main  {
+        margin: 0;
+        padding: 0;
+        min-height: 100vh;
+        font-family: var(--font);
+        background-color: silver;
+    }
+</style>
```
Comment (DiD3n) [inline .storybook/preview-head.html]: are there any way to import already existing index.css?

File: package.json [hunk 4]
```diff
@@ -88,6 +101,7 @@
     "@typescript-eslint/eslint-plugin": "^4.31.2",
     "@typescript-eslint/parser": "^4.31.2",
     "axios": "^0.23.0",
+    "babel-loader": "^8.2.5",
```
Comment (DiD3n) [inline package.json]: i'm pretty sure it's webpack loader

**Review by DiD3n — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (JUtupe) [reply package.json]: removed (unused anyway)

File: .storybook/preview-head.html (hunk 3 above)
Comment (JUtupe) [reply .storybook/preview-head.html]: good idea, fixed

File: .storybook/main.js (hunk 2 above)
Comment (JUtupe) [reply .storybook/main.js:31]: vite cool vite good

File: .github/workflows/storybook.yml (hunk 1 above)
Comment (JUtupe) [reply .github/workflows/storybook.yml:3]: changed to develop (I'll create this branch later)

**Pull request merged**